tests/test_elcometer456_block.py

```diff
@@ -20,6 +20,6 @@ def test_default_read(self):
         blk.stop()
         self.assertDictEqual(
             self.last_notified[DEFAULT_TERMINAL][0].to_dict(), {
-            "bytes": b"sample response"
+            "reading": "3.14"
```

Comment:
It would make more sense for this to be a float instead of a string.

We also often go with `value` as the name of our attribute names on signals. If `reading` doesn't have any special meaning here, I would go with that but either way works as long as it's documented.